style.css

```diff
@@ -432,6 +433,45 @@ textarea::placeholder {
   float: right;
 }
 
+/* Modal Projects */
+#modalContentProject {
+  background-color: #fff;
+}
+
+.modalBody {
+  max-height: 70vh;
+  grid-row: unset;
+}
+
+.modalHead {
+  display: flex;
+  justify-content: space-between;
+}
+
+.modal-tag {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #3c3a39;
+}
+
+.modal-description {
+  margin: 5% 0;
+}
+
+.modalButtons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.modal-button {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
```

Comment:
Great job implementing the details popup window, in order to ensure a better UI and good user interaction please implement the following changes for the mobile popup :

- [ ] Please use the right font type for the `heading : Multi post stories`, which is `font-type:roboto`. Also, use the right font color property which is `color: #172B4D;`
- [ ] kindly make the popup to occupy the full screen height because it looks doesn't occupy the whole screen hence some features are not properly implemented or visible 😄 

Figma design :
![image](https://github.com/rubydevi/my-portfolio/assets/20537712/c966a914-8531-42ee-b1dc-cc1ae119e3cb)

Your design:
![image](https://github.com/rubydevi/my-portfolio/assets/20537712/dcade9e8-9614-410d-96a5-108bc6870907)